webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -0,0 +1,26 @@
+export let pluginEditAttributes = {
```

Comment:
@tbrisker I am having trouble getting this method to work.. Can ya paste me a more complete example.